UPGRADE-1.10.md

```diff
@@ -1,4 +1,10 @@
+# UPGRADE FROM `v1.10.12` TO `v1.10.13`
+
+1. The support for Symfony 5.2 has been dropped, because it is not maintained version anymore. The recommended Symfony version 
```

Comment:
I would also add information about security issues - it is primary reason why we drop it :)
<img width="353" alt="Screenshot 2022-04-07 at 12 38 13" src="https://user-images.githubusercontent.com/17534504/162180887-bec0817d-88e3-4455-9742-f31f9140bb06.png">

Reply:
I would say the combination of these reasons (not maintained + not secured) is definitely a good reason to not test something 👌